libs/knowledge-store/ragstack_knowledge_store/langchain/cassandra.py

```diff
@@ -0,0 +1,194 @@
+from typing import (
+    Any,
+    Iterable,
+    List,
+    Optional,
+    Type,
+)
+
+from cassandra.cluster import ResponseFuture, Session
+from langchain_community.utilities.cassandra import SetupMode
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+
+from .base import KnowledgeStore, Node, TextNode
+from ragstack_knowledge_store import knowledge_store
+from ragstack_knowledge_store.edge_extractor import EdgeExtractor
+from ragstack_knowledge_store.embedding_model import EmbeddingModel
+
+
+class EmbeddingModelAdapter(EmbeddingModel):
```

Comment:
Should this be `_EmbeddingModelAdapter`? IIUC this isn't meant for public consumption.